docs/architecture/cluster-services.rst

```diff
@@ -23,41 +22,40 @@ metrics collection, tracing, node health checking and more.
 .. _Kops: https://github.com/kubernetes/kops/
 .. _Amazon AWS: https://aws.amazon.com
 
-In MetalK8s_, we augment a basic Kubernetes cluster deployed using the
-Kubespray_ playbook) with various tools to bring an on-premise cluster to the
-same level of operability.
+MetalK8s_ augments a basic Kubernetes cluster deployed using the
+Kubespray_ playbook with tools that bring an on-premise cluster to
+the same level of operability.
 
 .. _MetalK8s: https://github.com/scality/metal-k8s/
 .. _Kubespray: https://github.com/kubernetes-incubator/kubespray/
 
-Basic Cluster Addons
+Basic Cluster Add-ons
 --------------------
-On top of the basic Kubernetes services, the following addons are deployed:
+The following add-ons are deployed atop the basic Kubernetes services:
 
 Helm / Tiller
 *************
-Helm_ is a *package manager* for Kubernetes. It can be used to deploy various
-services in a Kubernetes cluster using templates to describe objects. *Tiller*
-is a cluster-side service used by the :command:`helm` CLI tool to manage these
+Helm_ is a *package manager* for Kubernetes that can deploy various services
+in a Kubernetes cluster using templates to describe objects. *Tiller* is a 
+cluster-side service the :command:`helm` CLI tool uses to manage these
 deployments.
 
 .. _Helm: https://www.helm.sh
 
 Heapster
 ********
-Heapster_ is a service which collects and exposes resource consumption metrics
-of containers running in a cluster. The Kubernetes Dashboard uses the Heapster
-service, when available, to display CPU and memory usage of Pods, Deployments
-and more.
+Heapster_ is a service that collects and exposes resource consumption metrics
+of containers running in a cluster. When the Heapster service is available, the
+Kubernetes Dashboard uses it to display CPU and memory usage of pods,
```

Comment:
Pod, Deployment and others are very specific object types in K8s, hence capitalization. I'd like to keep it that way, maybe should even be emphasized as well, or become terms in a glossary.

Reply:
I am aware that Google does it this way. Google, however, is wrong. The mere fact that they consider their object types somehow special doesn't give them the right to arrogate proper-noun status to them. 

This said, there is some logic in letting the Wookiee win. I'm afraid you're going to have to bear with me on this -- roll back the corrections, but keep vigilant: from time to time, you will continue to catch me reverting to English over Googlese.